Gemfile

```diff
@@ -55,7 +55,7 @@ group :test do
   gem "chromedriver-helper"
   gem "email_spec"
   gem "selenium-webdriver"
-  gem "shoulda-matchers"
+  gem "shoulda-matchers", "~> 4.0"
```

Comment:
My preference would be to leave gems un-version locked, and only lock them when we need to restrict ourselves to certain version.

Reply:
sure. carry over from upgrading to avoid deprecation warning there. I'll update.